pkg/operator/sync.go

```diff
@@ -138,7 +140,7 @@ func (optr *Operator) syncMachineConfigController(config renderConfig) error {
 		return err
 	}
 	sa := resourceread.ReadServiceAccountV1OrDie(saBytes)
-	_, _, err = resourceapply.ApplyServiceAccount(optr.kubeClient.CoreV1(), sa)
+	_, _, err = resourceapply.ApplyServiceAccount(optr.kubeClient.CoreV1(), nil, sa)
```

Comment:
we need the recorder here and wherever we're now using library-go. Look at https://github.com/openshift/cluster-kube-controller-manager-operator/blob/a01770cde88d646d3dfaa82a16bc8950afc7bb04/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L52-L77

we need basically the very same in the operator object/instance and just pass it here `optr.recorder`

Reply:
This is gonna panic otherwise